src/FondOfSpryker/Zed/Jellyfish/Business/Api/Adapter/OrderAdapter.php

```diff
@@ -2,19 +2,38 @@
 
 namespace FondOfSpryker\Zed\Jellyfish\Business\Api\Adapter;
 
+use FondOfSpryker\Zed\Jellyfish\Dependency\Service\JellyfishToUtilEncodingServiceInterface;
+use FondOfSpryker\Zed\Jellyfish\JellyfishConfig;
+use GuzzleHttp\ClientInterface;
 use Psr\Http\Message\ResponseInterface;
 use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
 
 class OrderAdapter extends AbstractAdapter
 {
     protected const ORDERS_URI = 'standard/orders';
 
+    /**
```

Comment:
Warum nutzt du nicht den Constructor in Abstract Class? Würde das direkt im Abstract übernehmen.